### PR TITLE
Actions/cache release 3.0.9

### PR DESCRIPTION
**Pull request by tiwarishub**
PR to release new version of actions/cache

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -28,4 +28,7 @@
 
 ### 3.0.8
 - Fix zstd not working for windows on gnu tar in issues [#888](https://github.com/actions/cache/issues/888) and [#891](https://github.com/actions/cache/issues/891).
-- Allowing users to provide a custom timeout as input for aborting download of a cache segment using an environment variable `SEGMENT_DOWNLOAD_TIMEOUT_MIN`. Default is 60 minutes.
\ No newline at end of file
+- Allowing users to provide a custom timeout as input for aborting download of a cache segment using an environment variable `SEGMENT_DOWNLOAD_TIMEOUT_MIN`. Default is 60 minutes.\
```
Comment (pdotl) [inline RELEASES.md:31]: There is a visible backslash at the end that can be removed.

Comment (tiwarishub) [reply RELEASES.md]: done

**Review by pdotl — APPROVED** [review]
LGTM 🚀

**Review by tanuj077 — APPROVED** [review]
LGTM

**Pull request merged**